### DISCUSSION TITLE
Add raid logs on the website

### DISCUSSION THREAD

**Discussion by djobf** [chromiecraft/chromiecraft#4079] [Suggestions]
You might be aware that the site legacyplayers.com shut down recently. It was buggy but it was still immensely useful for many guilds.
The dev published its [source code](https://www.reddit.com/r/wowservers/comments/vuli64/legacyplayers_will_go_offline_at_the_end_of_august/) stating that there was an API for pservers staff to integrate directly.
It's probably a lot of work but might be worth looking into as a great QoL improvement.

**Comment by Cooties-cc** [comment]
Not to bring up other servers, but I know Tauri had this system when they did their wrath server, Crystalsong. It would automatically track players in raids as they did bosses, and post data per boss to their website for others to check. It was a very interesting tool to see, because you could track some data otherwise not seen without an addon. While this was a great tool for players to understand and compare themselves to other classes, some also didn't like the tool cause it provided data they otherwise wanted private, or not shared because it caused animosity between players. Since the data cant be scrubbed, or a player to opt out of the collection, it could track things like deaths during the dance in naxx. If someone kept repeatedly dying to the dance, now instead of it being funny in the guild raid, the _entire server_ now knows you suck at the game. Same about DPS, and if you keep performing really badly, it could affect how others may treat you. 

Ofc this is extremes to the normal, and shouldn't be a core determining factor. I think more if the server can handle the data load, and it doesn't eat away at devs time to progress the server, it should 100% be a consideration to add to the server.

**Comment by Protandina** [comment]
Just implementing it like legacyplayers had it would be great benefit as it would spark some guild competition.

**Comment by Amrx2** [comment]
Hi @djobf, you can find it under the 'Tools' section on our website - https://legacyplayers.chromiecraft.com/

Thanks,
Construct
